catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/builder/XmlModelBuilder.java

```diff
@@ -218,6 +222,13 @@ public XmlModelBuilder setValue(String value) {
     return this;
   }
 
+  public XmlModelBuilder setDistance(Double distance) {
+    verifyResultNotYetRetrieved();
+    verifyTerminalNodeInProgress();
+    supplierInProgress = new DistanceBufferSupplier(supplierInProgress).setDistance(distance);
+    return this;
+  }
```

Comment:
❓ Should we add a unit test covering this case in `XmlModelBuilderTest`?

Reply:
added